sklearn/ensemble/gradient_boosting.py

```diff
@@ -2041,6 +2041,20 @@ class GradientBoostingClassifier(ClassifierMixin, BaseGradientBoosting):
     split. To obtain a deterministic behaviour during fitting,
     ``random_state`` has to be fixed.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import make_classification
+    >>> from sklearn.ensemble import GradientBoostingClassifier
+    >>> from sklearn.model_selection import train_test_split
+    >>> X, y = make_classification()
+    >>> X_train, X_test, y_train, y_test = train_test_split(X, y)
+    >>> clf = GradientBoostingClassifier()
+    >>> clf.fit(X_train, y_train)
+    GradientBoostingClassifier()
+    >>> clf.predict(X_test)
+    array([...])
+    >>> clf.score(X_test, y_test)  # doctest: +SKIP
```

Comment:
why skip doctest? maybe pass a random seed to train_test_split for the scores to be deterministic

Reply:
thank you for your feedback. made both examples deterministic.